src/netCDF4/__init__.pyi

```diff
@@ -624,6 +625,8 @@ def chartostring(
 ) -> npt.NDArray[np.str_] | npt.NDArray[np.bytes_]: ...
 
 def getlibversion() -> str: ...
+def rc_get(key: str) -> str: ...
```

Comment:
I think this signature should be
```suggestion
def rc_get(key: str) -> str | None: ...
```
and a null pointer check is necessary

Reply:
I don't see where it should return None.

Reply:
rc_get now checks for null pointer, returns None